### PR TITLE
Switch to using @blitz/installer helper functions …

### PR DESCRIPTION
**Pull request by kevinlangleyjr**
### What are the changes and their implications?
Switch to using `@blitz/installer` helpers instead of duplicating logic in each recipe.

### Checklist

- [ ] Changes covered by tests (tests added if needed)
- [ ] Does this PR warrant documentation updates? If yes, open a second PR with doc changes to [blitzjs.com](https://github.com/blitz-js/blitzjs.com)

### CODE REVIEW

**Review by MrLeebo — DISMISSED** [review]

**Comment by flybayer** [comment]
Sweet, thanks @kevinlangleyjr! 

Can you resolve the one merge conflict?

File: recipes/emotion/index.ts [hunk 1]
```diff
@@ -124,7 +107,7 @@ export default RecipeBuilder()
     explanation: `Update the Babel configuration to use Emotion's plugin and preset to enable some advanced features.`,
     singleFileSearch: paths.babelConfig(),
     transform(program: Collection<j.Program>) {
-      addBabelPlugin(program)
+      addBabelPlugin(program, "@emotion")
```
Comment (wKovacs64) [inline recipes/emotion/index.ts:110]: Oops, I totally missed this in  #2156. Glad to see it'll be fixed here. 👍

Comment (kevinlangleyjr) [reply recipes/emotion/index.ts:110]: @wKovacs64 I saw your PR right after I submitted mine!

**Comment by kevinlangleyjr** [comment]
Merge conflict resolved @flybayer! 🙂  🚀

**Pull request merged**